snapshot/pkg/volume/openebs/processor.go

```diff
@@ -236,6 +240,7 @@ func (h *openEBSPlugin) SnapshotRestore(snapshotData *crdv1.VolumeSnapshotData,
 	volumeSpec.Metadata.Labels.PersistentVolumeClaim = pvc.ObjectMeta.Name
 	volumeSpec.Metadata.Name = pvName
 
+	glog.Infof("OldVol Passing pv '%s' and namespace ' %s", pvRefName, pvRefNamespace)
```

Comment:
Is this log message for debug purpose? If not, can you provide a sentence here that helps users with information.

Reply:
removed !

Reply:
debug log, removed !